packages/grid/_modules_/grid/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -340,4 +351,23 @@ export const useGridColumnResize = (
 
   useGridApiOptionHandler(apiRef, GridEvents.columnResize, props.onColumnResize);
   useGridApiOptionHandler(apiRef, GridEvents.columnWidthChange, props.onColumnWidthChange);
+
+  const componentsProps = React.useMemo<GridSlotsComponentsProps>(
+    () => ({
+      ...props.componentsProps,
+      columnsHeader: {
+        ...props.componentsProps?.columnsHeader,
+        resizeCol: resizingColumnField,
+      },
+    }),
+    [props.componentsProps, resizingColumnField],
+  );
+
+  return React.useMemo(
```

Comment:
Why to return props if they are not used in `useDataGridProComponent`?

Reply:
Oh I forgot to use them, thanks !